src/components/backup/BackupIcloudStep.js

```diff
@@ -256,14 +257,18 @@ const BackupIcloudStep = () => {
         dispatch(setIsWalletLoading(null));
       },
       password,
-      walletId: params?.walletId,
+      walletId,
     });
+    setTimeout(() => {
```

Comment:
Same as comment in BackupConfirmPasswordStep with regard to walletCloudBackup swallowing errors and then calling onError

Reply:
Resolved in https://github.com/rainbow-me/rainbow/pull/1003